jsonlogging/src/main/java/org/secondbase/logging/JsonLoggerModule.java

```diff
@@ -0,0 +1,30 @@
+package org.secondbase.logging;
+
+import org.secondbase.core.SecondBase;
+import org.secondbase.core.config.SecondBaseModule;
+import org.secondbase.flags.Flag;
+
+public class JsonLoggerModule implements SecondBaseModule {
+
+    @Flag(name = "environment",
+            description = "The environment the service runs in (test|staging|production).")
+    private static String environment = "test";
+
+    @Flag(name = "service", description = "The name of the service (testservice).")
+    private static String service = "testservice";
+
+    @Flag(name = "datacenter",
+            description = "The datacenter the service runs in (eu-west-1|ap-southeast-1).")
+    private static String datacenter = "eu-west-1";
+
```

Comment:
Yeah, that would currently be possible but the same limitation I mentioned before applies. Where do we call that? Anyway, I think this is something that we should decide at a later point in time in the spirit of not letting this pr getting out of hand.